src/ui/textfield.rs

```diff
@@ -182,7 +191,7 @@ impl TextField {
                 let selection_anchor_text: String = self.text[..self.selection.start()].iter().collect();
                 let selection_anchor_text_width = ui.borrow_font().measure_str(selection_anchor_text, None).0;
 
-                // Get text left to right or right to left depending on end cursor position.
+                // Get text selected text
```

Comment:
There's still a minor typo in this comment, I think it should say "Get selected text".